dashboard/app/helpers/survey_results_helper.rb

```diff
@@ -19,10 +20,9 @@ def show_diversity_survey?(kind)
   end
 
   def show_nps_survey?
-    return false unless SurveyResultsHelper::NPS_SURVEY_ENABLED
+    return false unless NPS_AUDIENCE != "none"
```

Comment:
nit: use the constant here? (it might need to be renamed to `NPS_AUDIENCE_NONE` or something but that's up to you)